packages/patternfly-4/react-virtualized-extension/package.json

```diff
@@ -40,7 +40,6 @@
   },
   "peerDependencies": {
     "@patternfly/react-table": "^2.10.0",
-    "lodash-es": "4.x",
```

Comment:
should `lodash` be added as a `peerDependency` now?

Reply:
`react-virtualized-extension` doesn't use `lodash-es` anywhere.